### ISSUE TITLE
Invisible Items with Open Security

### ISSUE THREAD

**Issue by eulerlobo** [ganymedes01/TheErebus#745]
### Erebus version
Version: 1.0.30

### Forge version
Version: 14.23.5.2838

### Expected behavior
The follow items should be visible:
 - Bamboo Shield `<erebus:bamboo_shield>`
 - Exoskeleton Shield `<erebus:exoskeleton_shield>`
 - Jade Shield `<erebus:jade_shield>`
 - Reinforced Exoskeleton Shield `<erebus:rein_exoskeleton_shield>`
 - Wasp Sword `<erebus:wasp_sword>`
 - Wasp Dagger `<erebus:wasp_dagger>`
 - Augmented Scorpion Pincer `<erebus:enhanced_scorpion_pincer>`
 - Quake Hammer `<erebus:war_hammer>`
 - Web Slinger `<erebus:web_slinger>`
 - Wither Web Slinger `<erebus:web_slinger_wither>`
 - Wand of Animation `<erebus:wand_of_animation>`
 - Staff of Gaea `<erebus:portal_activator>`
 - Wand of Preservation `<erebus:wand_of_preservation>`

### Actual behaviour
The above items render invisible with OpenSecurity.

### Steps to reproduce the problem
1. Install the Erebus, [version 1.0.30](https://www.curseforge.com/minecraft/mc-mods/the-erebus/files/2727867), with the OpenComputer, [version 1.7.4.153](https://www.curseforge.com/minecraft/mc-mods/opencomputers/files/2670112) (the last version in CurseForge) and OpenSecurity, [version 1.0-35](https://www.curseforge.com/minecraft/mc-mods/opensecurity/files/2756561) (the last version in CurseForge).
2. Install JEI for help search the items, but is optional.
3. Don't need to change any configuration in the files from the three mods, just start or load a game and the above items will render invisible.

### Observation
Maybe is the same problem of issue #734. And I don't know if I need to open a Issue in the GitHub of [OpenSecurity ](https://github.com/PC-Logix/OpenSecurity/issues)also, do I?

### Print Screen
![JEI](https://i.imgur.com/Mh44rYJ.png)

**Comment by Deliphin** [comment]
Can confirm, I have a modpack with the following:
```
OpenComputers-MC1.12.2-1.7.4.153
OpenSecurity-MC1.12.2-1.0-35
Erebus-1.0.30
```
If I remove OpenSecurity and nothing else, Erebus starts working correctly.

~~Also, I can't place the staff of gaea onto the keystone when OpenSecurity is installed.~~ My portal was set up incorrectly, I had a leaves block where it didn't belong.

**Comment by ben-mkiv** [comment]
Bug was in OpenSecurity, so this can be closed. (Fixed with OpenSecurity version 1.0-39)

Anyways, we should both rely on `Item#setTileEntityItemStackRenderer` instead of overriding `TileEntityItemStackRenderer.instance`

**Comment by vadis365** [comment]
fixed as of above merged.
